rj_gameplay/stp/role/cost.py

```diff
@@ -0,0 +1,83 @@
+"""This module contains a variety of functions that return cost functions for
+convenience."""
+
+from typing import Optional
+
+import numpy as np
+import stp.rc as rc
+import stp.role as role
+
+
+def flat_switch_cost(
```

Comment:
Previous comments on this still apply.

Reply:
wdym?

Reply:
You marked this comment chain: https://github.com/RoboJackets/robocup-software/pull/1577/files/b8e6f7053e87e223dd7a2a5005eeda4799a7708d#r491723289 as resolved. My concern mentioned is still not addressed (i.e. I don't like trying to saturate costs).

Reply:
You're looking at old code. Check the Files Changed tab.